catalyst_voices/utilities/uikit_example/devtools_options.yaml

```diff
@@ -0,0 +1,3 @@
+description: This file stores settings for Dart & Flutter DevTools.
+documentation: https://docs.flutter.dev/tools/devtools/extensions#configure-extension-enablement-states
```

Comment:
Do we need this file?

Reply:
We don't. My ide generated them automatically i remove them.